db/models/news.model.js

```diff
@@ -16,3 +16,19 @@ exports.fetchArticleById = (id) => {
       return rows[0];
     });
 };
+
+exports.updateArticleById = (id, newVote) => {
+  return db
+    .query(
+      `UPDATE articles
+       SET
+       votes = votes + $2
```

Comment:
I'd recommend swapping the $1 and $2 around here so they appear in the order that they are being used.

Reply:
Done! Should I do another PR?